### DISCUSSION TITLE
Toggle File Change Turn on

### DISCUSSION THREAD

**Discussion by ghost** [gitkraken/vscode-gitlens#2490] [Q&A]
Hi,
  Could it possible that turn on the Toggle File Change by default.
  And turn on for all the files.
  And disable to use Esc to turn off that ?

  It is useful for me so so much.
  Thanks a lot.

**Comment by hflsmax** [comment]
Second this. I would like to keep "Toggle File change" on while I edit the file.

Currently, the feature is immediately turned off once I edit the file

**Comment by rcketscientist** [comment]
And bring back the gutter diff insights?  These changes were a UX nightmare.

**Comment by SIGSTACKFAULT** [comment]
Yes please! I like it on all the time and i use vim keybindings so i press esc constantly.

**Comment by CassandraCat** [comment]
I want this, I'm using Vim, and canceling with the Esc key makes me feel very uncomfortable.

**Reply by eamodio** [reply]
You can now toggle this off:
![image](https://github.com/gitkraken/vscode-gitlens/assets/641685/7a9ab74c-07da-40cb-b279-4dae997332ae)

e.g. set `"gitlens.fileAnnotations.dismissOnEscape": false`

**Reply by CassandraCat** [reply]
Solved it!
